frontend/package.json

```diff
@@ -7,7 +7,7 @@
     "test:unit": "vue-cli-service test:unit",
     "build": "vue-cli-service build",
     "lint": "vue-cli-service lint",
-    "license-check": "license-checker --production --json --onlyAllow 'MIT;Python-2.0;Apache-2.0;BSD-2-Clause;BSD-3-Clause' --excludePackages 'company-agent-frontend@0.1.0'",
+    "license-check": "license-checker --production --json --onlyAllow 'MIT;Python-2.0;Apache-2.0;BSD-2-Clause;BSD-3-Clause;LGPL-3.0;ISC;' --excludePackages 'company-agent-frontend@0.1.0'",
     "license": "generate-attribution -o ./licenses",
```

Comment:
Can we set this as a dev dependency? Otherwise we have to jump through a lot of hoops to make this work. Currently we only have the obligation to provide the respective license text files, which we do via the license container. This is the first license that also requires to provide the source code. See also:

https://github.com/hyperledger-labs/business-partner-agent/blob/master/Dockerfile.license
And the attribution steps in
https://github.com/hyperledger-labs/business-partner-agent/blob/master/.github/workflows/build.yml

Reply:
I'll look into making it dev dependency only.  makes sense.

Reply:
This only excludes the module from the check, but the module is still imported. As the license states 


> Can I use LAME in my commercial program?  
> 
> Yes, you can, under the restrictions of the LGPL.  The easiest
> way to do this is to:
> 
> 1. Link to LAME as separate jar (lame.min.js or lame.all.js)
> 
> 2. Fully acknowledge that you are using LAME, and give a link
>    to our web site, lame.sourceforge.net
> 
> 3. If you make modifications to LAME, you *must* release these
>    these modifications back to the LAME project, under the LGPL.

and the license text is displayed in the About section we could argue that we fulfill the license as the link is displayed and we did not modify any sources.

Reply:
What's missing in the list is the need to provide the "complete corresponding source code" or is that contained in the jar, anyway?

Reply:
I'm no JS expert by I think the setup in the package.json is wrong. When I installed locally with `npm install --no-optional` the dependency was still referenced. It might be because it is defined twice in https://github.com/antoine92190/vue-advanced-chat/blob/0.8.9/package.json 

To make this work I had to set it as optional in our package json:

```json
  "optionalDependencies": {
    "lamejs": "^1.2.0"
  }
```
Then install without optional dependencies worked. So i guess we need three changes
1. Reset the --excludedPackages to as they where before
2. Add the optional dependencies like above
3. add the no optional flag to our build pipeline

Then the lame decoder is not a production dependency, but you can still use it locally in dev mode if needed. And we can leave the license list unchanged.

Reply:
Looks like lamejs is not really optional, and the project does not compile any more if lamejs is missing, could also be that I did something wrong. Anyway, I added a ticket in the vue-advanced-chat repo to clarify this: https://github.com/antoine92190/vue-advanced-chat/issues/190